main.py

```diff
@@ -2,6 +2,55 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    # 1. Create:
+    genres_for_create = [
+        "Western",
+        "Action",
+        "Dramma"
+    ]
+    actors_for_create = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+
+    for genre in genres_for_create:
+        Genre.objects.create(
+            name=genre,
+        )
```

Comment:
You are using a `for` loop to create Genre objects, which is good as it adheres to point #3 in the checklist.